app/AppKernel.php

```diff
@@ -54,7 +56,7 @@ public function registerBundles()
             // PrestaShop Core bundle
             new PrestaShopBundle\PrestaShopBundle(),
             // PrestaShop Translation parser
-            new PrestaShop\TranslationToolsBundle\TranslationToolsBundle(),
+            new TranslationToolsBundle(),
```

Comment:
seem not related to the PR ?

Reply:
I could not leave it as it is. Because of the Prestashop class name that create conflicts with this namespace.